bode/bode/models/tag.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+
+from sqlalchemy import ForeignKey
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.exc import NoResultFound
+
+from bode.app import db
+
+
+class Tag(db.Model):
+    __tablename__ = "tags"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    task_id = db.Column(
+        UUID(as_uuid=True), ForeignKey("tasks.id"), nullable=False
+    )
+    name = db.Column(db.String(80), unique=True, nullable=False)
+
+    task = db.relationship("Task", back_populates="tags")
+
+    __table_args__ = (
+        db.UniqueConstraint("task_id", "name", name="_task_id_name_uc"),
+    )
+
+    def create(**tag_data):
+        tag = Tag(**tag_data)
+
+        db.session.add(tag)
+        db.session.commit()
+
+        return tag
+
+    def delete(tag_id):
+        tag = Tag.query.get(tag_id)
+
+        if tag is None:
+            raise NoResultFound
```

Comment:
Protip: you can use `Tag.query.get_or_404(tag_id)` to omit the if and raise:
```suggestion
        tag = Tag.query.get_or_404(tag_id)
```